bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -117,6 +118,39 @@ def add_hofers(filename, data):
         )
 
 
+def parse_cve_id(cve_id):
+    cve_year, cve_order = cve_id.split('-')[1:]
+    return int(cve_year), int(cve_order)
+
+
+def add_or_update_cve(data):
+    for cve_id, advisory in data['advisories'].iteritems():
+        cve_year, cve_order = parse_cve_id(cve_id)
+        update_advisory_bugs(advisory)
+        cve_title = advisory.get('cve_problemtype', advisory.get('title')) or ''
```

Comment:
@dveditz I've just pushed this change. You can add a `cve_problemtype` field to any advisory in the `advisories` list and it will override the title for that one in the CVE feed. We can always tweak later if need be.